src/atoms/buttonLink.js

```diff
@@ -0,0 +1,87 @@
+import React from 'react';
```

Comment:
also, the atom folder should be a subfolder of `src/components`

Reply:
ok pour moi c'est différent ca les component ont une logique metiers, alors que les atoms sont purement fonctionnels